integration-tests/cli/test/execute-workflow.test.ts

```diff
@@ -199,3 +199,25 @@ test.serial(
     });
   }
 );
+
+test.serial(
```

Comment:
A test for a workflow that uses two different versions of the same Adaptor. 
We know it used both different versions if they're both resolved.